src/lib/bootloader/grub2_widgets.rb

```diff
@@ -1055,66 +1064,82 @@ def loader_location_widget?
       end
 
       def generic_mbr_widget?
-        (Yast::Arch.x86_64 || Yast::Arch.i386) && grub2.name != "grub2-efi"
+        (Yast::Arch.x86_64 || Yast::Arch.i386) && !["grub2-efi", "grub2-bls"].include?(grub2.name)
       end
 
       def secure_boot_widget?
-        Systeminfo.secure_boot_available?(grub2.name)
+        Systeminfo.secure_boot_available?(grub2.name) && grub2.name != "grub2-bls"
```

Comment:
Michael Chang has written:
---I don’t think we need "secure boot" since it’s specific to ordinary grub----
That's why I have not implemented. But we still can do it afterwards ;-)